### ISSUE TITLE
Synthesize data on website usage

### ISSUE THREAD

**Issue by AmyKort** [18F/fec-cms#1277]
In order to effectively plan for the strategic and stable growth of the website, we need to summarize a value proposition for the site as a whole. One step in this process is to review what we already know about who uses the website, how they use the site and how users have reported wanting to use the site. To complete this review, we'll look at

User personas
Analytics
Research phase interviews
Results from usability testing where appropriate

with an eye toward cataloging and synthesizing the site-wide priorities identified by our users.

@Cptechiegal

**Comment by jameshupp** [comment]
Brief outline of what we're pulling together:

1. User personas: what we know about our users and how they were using FEC.gov coming into this work.
2. Analytics: knowing the motivations, goals, needs, and frustrations of our users gives us direction. Understanding how/whether we succeed is measurable.
3. Usability research: analytics tells us what is happening. User-centered design—prototyping, testing, and interation; getting qualitative feedback and reacting to it—is how we succeed.

What we are trying to get to with this is that both analytics and usability research are essential. This isn't necessarily going to let us draw firm conclusions going into the value proposition work, since without establishing that value it's hard to be sure we're measuring the right things. But we'll be able to articulate the relationship among the different ways of measuring.

**Comment by dorothyyeager** [comment]
Thanks @jameshupp  - very helpful!

**Comment by jameshupp** [comment]
Here's where we are. @JonellaCulmer would love your feedback!
🔒  https://docs.google.com/document/d/1B0Er_qZ4o6fQzQiI51u8p5XRN9M4uQLVt4pJdw-Bi2w/edit# 🔒 

I think there are additional lines we need in here about one or two other kinds of research to really flesh out the idea of user-centered design. And (noting this here just for my own sake) we still need to incorporate a link back to the User Centered Resource Guide. The usability research section in particular is also totally underdeveloped. But I'm trying not to over-belabor any of this.

**Comment by AmyKort** [comment]
This is great! As a second step, I think folks on the content team need to spend some time with the primary resources, using the document as a framework for how to interpret them. As part of our transition, we need to understand the data we have and how to interpret it on our own, and I don't know how to do that other than by practicing. While we can't necessarily draw conclusions for the value proposition from our analytics, I was hoping we'd be able to look at some of the early research and interviews with users--or the readouts from those interviews--so we'd be able to better understand and articulate what we've learned about users' hopes and pain points. That would go a long way toward making us better advocates for our users.

Should I open a separate issue for compiling the data sets we have in one place where content team folks can read them?

**Comment by dorothyyeager** [comment]
That would be great, Amy.

**Comment by AmyKort** [comment]
On second thought, let's carry this issue into the next sprint since there's so much great information here for us to reference. 

For next steps, let's review what this information tells us about who are we trying to serve with the website. For example, in the past, analytics have told us that data users are by far the largest user group for the website.  FEC has other functions that are important to communicate via a website, and we need to look at those too. This exercise will help inform our value proposition.
- [ ] Content team planning meeting
- [ ] 	Content team reviews personas and other initial research to determine what users want/need from FEC website
- [ ] 	Read out on this portion of data
- [ ] 	Identify any internal needs 
- [ ] 	Team review research 
- [ ]  Full read out

**Comment by AmyKort** [comment]
I've looked through the personas, draft principles and other initial research we've compiled and put together some notes about what we've learned from site users so far. The big piece that is missing from these notes is information from our analytics. But, any comments on the draft so far would be awesome. I'm hoping we can use this during the facilitation meetings so that we have some information in front of us and something to react to during our discussions. Thanks!

https://docs.google.com/document/d/1a8fK4_W3x5MeQiCxUXoEQLWB76_lh9avRD5yOuWRrHQ/edit#
